tests/wpt/metadata/workers/semantics/interface-objects/001.worker.js.ini

```diff
@@ -1,5 +1,6 @@
 [001.worker.html]
   type: testharness
+  expected: TIMEOUT
```

Comment:
Apparently this times out in a debug build - please manually delete this line, and these two lines:
```
  [The ImageBitmap interface object should be exposed.]
    expected: FAIL
```

Reply:
The checks fail upon removing the above lines.